src/Templates/AdminPanel/Bit.AdminPanel/Web/Pages/Categories/CreateEditCategoryPage.razor

```diff
@@ -6,42 +6,48 @@
 <PageTitle>Create/Edit category (@Category?.Name)</PageTitle>
 
 <div class="page-container">
-    <EditForm Model="Category" OnValidSubmit="WrapHandled(Save)" class="form--white">
-        @if (IsLoading)
-        {
-            <div class="spinner">
-                <BitSpinner></BitSpinner>
-            </div>
-        }
-        <DataAnnotationsValidator />
-        <h1 class="form-title">Category</h1>
-        <div class="form-input-container">
-            <BitTextField Label="@(nameof(Category.Name))" @bind-Value="Category!.Name" />
-            <ValidationMessage For="() => Category.Name" />
+    <div class="page-content--white">
+        <div class="page-header">
+            <h3>Create/Edit category</h3>
         </div>
-
-        <div class="form-input-container">
-            <label for="catColorInput">Color</label>
-            <BitColorPicker id="catColorInput" ShowPreview="true" @bind-Color="Category!.Color">Default Color Picker</BitColorPicker>
-            <ValidationMessage For="() => Category.Color" />
+        <div class="content align-center">
```

Comment:
please don't use a generic name that describes the technicality of the style of a CSS class applies (like this `align-center`). a better name would be `form-container` here.

and what is this `content` class?